content/en/examples/controllers/nginx-deployment.yaml

```diff
@@ -19,3 +19,8 @@ spec:
         image: nginx:1.14.2
         ports:
         - containerPort: 80
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
```

Comment:
Let's keep the initial example simple. To illustrate a rolling deployment, make a new file and introduce that later. For example, within https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#writing-a-deployment-spec

Reply:
I would actually use [tabs](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#tabs) and have one example for `maxSurge` and another for `maxUnavailable`. Maybe a third tab where you use both, perhaps named “Hybrid”.

Reply:
Thanks @sftim. I have understood the changes. Will update this PR accordingly.